packages/api-clients/open-api/bffApi.yml

```diff
@@ -15449,6 +15442,21 @@ paths:
               format: uuid
             default: []
           explode: false
+        - in: query
+          name: offset
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 0
+        - in: query
+          name: limit
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            maximum: 50
```

Comment:
is the frontend aligned with this change?

Reply:
Yup!